main.cpp

```diff
@@ -73,12 +25,27 @@ void testPairToNumber(
     assert(pairNumber == expectedPairNumber);
 }
 
+void ToString() {
+    cout<<"Number to Color Mapping"<<endl;
+    cout<<"Pair Number"<<"\t"<<"Major Minor"<<endl;
+    
+    int pair = 1 ;
+    while(pair <= 25){
```

Comment:
Don't hardcode the loop limit. It's easy to forget updating this when adding or removing colors.